book/whats_new/20200119_plugiun_accounts.md

```diff
@@ -0,0 +1,31 @@
+# Plugin accounts
+
+This article will cover the changes we made to the access for plugin developers
+
+## Main Changes
+
+Each plugin has an owning account.
+When submitting a new plugin using zappifest, use the --account with a valid account id:
+
+zappifest publish --manifest <path-to-manifest-json-file> --access-token <zapp-access-token> --account <account-id> --new
```

Comment:
zappifest publish --manifest <path-to-manifest-json-file> --access-token <zapp-access-token> --account <account-id> --new

should be 

```
zappifest publish --manifest <path-to-manifest-json-file> --access-token <zapp-access-token> --account <account-id> --new
```